bin/webpack.js

```diff
@@ -399,6 +405,12 @@
 				}
 			});
 
+			ifArg("info-verbosity", function(value) {
+				if (typeof value !== "string" || !["none", "info", "verbose"].includes(value))
+					throw new Error("Invalid configuration object. \n configuration['info-verbosity'] should be one of these:\n \"none\" | \"info\" | \"verbose\"");
```

Comment:
I think we've got a error helper for this

Reply:
I maybe misunderstood, but couldnt find the usage in codebase of error helper, could you please point me into the right direction here? All i've found is `bin/errorHelpers.js` which basically works with existing error but doesnt throw one for me

Reply:
Hm, okay, so if you look in the function inside the bin folder that I believe is called convert-argv, there's a processing of the options, and if it's not matching the schema, it throws. At least that is what I think it's doing under the hood. I'm not on a computer right now, so let me get back to you in a bit

Reply:
yeah but seems it doesnt support enum which could substitue `![].includes` call as jsonschema does?

Reply:
It should only be a string